src/server/scripts/EasternKingdoms/ZulGurub/boss_jeklik.cpp

```diff
@@ -16,19 +16,27 @@
  */
 
 #include "GameObjectAI.h"
+#include "MoveSplineInit.h"
 #include "ScriptMgr.h"
 #include "ScriptedCreature.h"
+#include "SmartAI.h"
 #include "SpellScript.h"
 #include "TaskScheduler.h"
+#include "WaypointMgr.h"
 #include "zulgurub.h"
 
-enum Says
+enum BossSays
 {
     SAY_AGGRO                   = 0,
     SAY_CALL_RIDERS             = 1,
     SAY_DEATH                   = 2,
     EMOTE_SUMMON_BATS           = 3,
-    EMOTE_GREAT_HEAL            = 4
+    EMOTE_GREAT_HEAL            = 4,
```

Comment:
```suggestion
    EMOTE_GREAT_HEAL            = 4
```
Comma added unnecessarily?